### PR TITLE
Specify user import

### PR DESCRIPTION
**Pull request by louischan-oursky**
ref #3517

### CODE REVIEW

File: docs/specs/user-import.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+  "id": "some_opaque_string",
+  "created_at": "2024-01-01T00:00:00.000Z"
+  "status": "completed",
+  "summary": {
+    "total": 100,
+    "inserted": 50,
+    "updated": 49,
+    "failed": 1
+  },
+  "errors": [
```
Comment (tung2744) [inline docs/specs/user-import.md:161]: Can we include the item index here, so user can easily find the item?

**Comment by louischan-oursky** [comment]
Note to myself: Add warnings for IGNORED attributes.

File: docs/specs/user-import.md (hunk 1 above)
Comment (louischan-oursky) [reply docs/specs/user-import.md]: JSON path seems overkill. I added `index` instead.

**Comment by louischan-oursky** [comment]
Removed support for plain password. The rationale is, if they know the plain password, they can easily compute the bcrypt hash of it. So taking plain password is unnecessary (and it is insecure)

**Comment by tung2744** [comment]
@louischan-oursky I guess we can merge this?

**Pull request merged**